src/configurations/destinations/smartly/ui-config.json

```diff
@@ -0,0 +1,203 @@
+{
+  "uiConfig": {
+    "baseTemplate": [
+      {
+        "title": "Initial setup",
+        "note": "Review how this destination is set up",
+        "sections": [
+          {
+            "groups": [
+              {
+                "title": "Connection settings",
+                "icon": "settings",
+                "fields": [
+                  {
+                    "type": "textInput",
+                    "label": "Api Token",
+                    "note": "",
+                    "configKey": "apiToken",
+                    "secret": true,
+                    "regex": "(^\\{\\{.*\\|\\|(.*)\\}\\}$)|(^env[.].+)|.*",
+                    "regexErrorMessage": "Invalid Api Token",
+                    "placeholder": "e.g: MOx2ZmMwLNE2A2IdNKL0N2VhN2XXXXX1MTVmMzA1ODk0YmIkNDZhNTojMTk3YTBlMTg1YXXXXXMDA2ZDVmZjY0ZGFiOTVkNDMXXXXcwOWFk"
```

Comment:
Is this placeholder contains secret keys ?

Reply:
No